### PR TITLE
Add config for pk50 pk100 pk200

### PR DESCRIPTION
**Pull request by pfanguin**
Test link : 
https://mf-geoadmin3.dev.bgdi.ch/?api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Ffap_pk_shop&topic=ech&lang=en&bgLayer=ch.swisstopo.pixelkarte-farbe&X=184627.62&Y=687027.62&zoom=0&layers=ch.swisstopo.pixelkarte-pk25.metadata,ch.swisstopo.pixelkarte-pk100.metadata,ch.swisstopo.pixelkarte-pk200.metadata,ch.swisstopo.pixelkarte-pk50.metadata&layers_visibility=true,false,false,false

### CODE REVIEW

**Comment by pfanguin** [comment]
This PR must be merged for today, we would like deploy on int the pixelkarte

File: chsdi/models/vector/stopo.py [hunk 1]
```diff
@@ -936,57 +936,61 @@ class GridstandPk25Meta(Base, GridstandTemplate, ShopStandardClass, Vector):
...
     __table_args__ = ({'autoload': False})
+    id = Column('bgdi_id', Integer, primary_key=True)
     the_geom = Column(Geometry2D)
 
 
-class GridstandPk200Meta(Base, GridstandTemplate, Vector):
+class GridstandPk200Meta(Base, GridstandTemplate, ShopStandardClass, Vector):
     __bodId__ = 'ch.swisstopo.pixelkarte-pk200.metadata'
     __tablename__ = 'view_gridstand_datenhaltung_pk200_tilecache'
-    __template__ = 'templates/htmlpopup/pk200_metadata.mako'
```
Comment (loicgasser) [inline chsdi/models/vector/stopo.py]: pk200_metadata.mako still is in the project? same question for the others

Comment (pfanguin) [inline chsdi/models/vector/stopo.py]: Ah yes, I forgot to remove the mako, sorry.

**Comment by loicgasser** [comment]
We have a lot data errors....
Not really related to this pr except for swissbuildings2 seems like there is no table...
@ltclm what's going with sphinx? geom_st_box2d missing some addresses change, `bav.schienennetz_segment.id` still missing, 
See: https://jenkins.dev.bgdi.ch/job/PR_chsdi3/53/console

**Comment by loicgasser** [comment]
I'll merge but we need to fix that. also geocat services are down??https://www.geocat.ch/geonetwork/srv/eng/csw?request=GetRecordById&elementSetName=full&outputFormat=application/xml&outputSchema=http://www.geocat.ch/2008/c%20he&service=CSW&version=2.0.2&id=055192b1-c378-4e66-8d43-c926f0be7980

https://www.geocat.ch/geonetwork/srv/eng/csw?request=GetRecordById&elementSetName=full&outputFormat=application/xml&outputSchema=http://www.geocat.ch/2008/c%20he&service=CSW&version=2.0.2&id=a4ecf3a4-9943-4826-a997-9d7523e4d8ad

What is the new link ??

**Comment by pfanguin** [comment]
Ok thanks

**Pull request merged**